images/runtime/php-fpm/8.1/base.bullseye.Dockerfile

```diff
@@ -76,10 +76,7 @@ RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
         xsl
 RUN pecl install redis && docker-php-ext-enable redis
 # https://github.com/Imagick/imagick/issues/331
```

Comment:
Nit: should we remove this comment from the template file? Similar comment for sqlsvr

Reply:
I'll update this in a follow-up PR (or if we need to make additional changes to this PR) if that is OK.

Reply:
Of course, thanks!